src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -1754,6 +1754,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return position >= start AndAlso position <= _end
         End Function
 
+        Public Function IsInPropertyDeclarationHeader(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsInPropertyDeclarationHeader
+            Dim propertyDeclaration = node.GetAncestor(Of PropertyStatementSyntax)()
+
+            If propertyDeclaration Is Nothing Then
+                Return False
+            End If
+
+            Dim start = If(propertyDeclaration.AttributeLists.LastOrDefault()?.GetLastToken().GetNextToken().SpanStart, propertyDeclaration.SpanStart)
+            Dim [end] = If(propertyDeclaration.AsClause?.FullSpan.[End], propertyDeclaration.Identifier.FullSpan.End)
```

Comment:
note: this looks slightly different than the code you removed.  specifically:

```vb
            If containingProperty.HasReturnType() AndAlso
                position > containingProperty.GetReturnType().Span.End Then
                Return Nothing
            End If
```

are the semantics the same?

Reply:
Yes, they're. GetReturnType() just calls `member.GetAsClause()`. I can move it to the helper as to be more clear.

Reply:
TBH the .GetAsClause seems to me more clear in this context. It obviously identifies the Tokens in question. And since we want to be clear what the header is it servers better IMO.

Reply:
i'm totally fine with using GetAsClause :)  i think it's clearer too. I just wanted to make sure there was no unintended change in behavior.